src/create_torrent.cpp

```diff
@@ -455,6 +455,12 @@ namespace {
 			aux::throw_ex<system_error>(errors::invalid_piece_size);
 		}
 
+		// this is an unreasonably large piece size. Some clients don't support
+		// pieces this large.
+		if (piece_size > 128 * 1024 * 1024) {
```

Comment:
Am I the only one who doesn't like to look for such fundamental limits among the code, rather than declared as constants in some more appropriate place?